src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -121,18 +121,23 @@ public void removeSession(Name name, Index sessionIndex) {
 
     /**
      * Returns student with {@code name}.
-     * @param name must be unique and exact.
-     * @return Student
+     * The student with the exact name must already exist in the address book.
      */
     public Student getStudentWithName(Name name) {
         requireNonNull(name);
         return students.getStudentWithName(name);
     }
 
+    /**
+     * Returns true if a student with this name exists in the unique student list
+     */
     public boolean hasName(Name name) {
         return students.hasName(name);
     }
 
+    /**
+     * Returns true if {@code session} exists in any of the students in the unique student list
```

Comment:
Not sure if capitalization of session here would make a difference since it is referring to the class Session in the javadocs.

Reply:
What do you think of making it like this?
`Returns true if session {@code Session} exists in any of the students in the unique student list`
I think this takes away the ambiguity of referring to either the session variable or the Session class.

Reply:
Same comment as above (regarding WHAT and WHY not HOW).